ql/src/test/queries/clientpositive/auto_sortmerge_join_17.q

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE tbl1_n5(key int, value string) CLUSTERED BY (key) SORTED BY (key) INTO 2 BUCKETS;
+
+insert into tbl1_n5(key, value)
+values
+(0, 'val_0'),
+(2, 'val_2'),
+(9, 'val_9');
+
+set hive.optimize.semijoin.conversion = false;
+
+explain
```

Comment:
Can we also add a negative test case where the number of group-by within a reducer is 1 or 0 and we expect to see the SMB join being used.

Reply:
There are lots of q tests for SMB joins with group by.
Example: https://github.com/apache/hive/blob/2eef89b5b6f350c2c9a72499c3253760df20a328/ql/src/test/results/clientpositive/llap/limit_pushdown.q.out#L1108-L1150

Reply:
Ok, we can close this comment.